lib/views/survey/survey_field.dart

```diff
@@ -0,0 +1,375 @@
+// ignore_for_file: avoid_renaming_method_parameters, type_annotate_public_apis
```

Comment:
I added these ignores since I wanted the different survey field builder functions to not be super long.